README.md

```diff
@@ -151,6 +151,29 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/t
 
 ## 🤝 Contributing <a name="contributing"></a>
```

Comment:
This section seems duplicated to the author section, please kindly remove one.

Reply:
Yes, good idea. Will resolve by next PR